README.md

```diff
@@ -6,7 +6,7 @@ description: High Performance Telemetry Agent for Logs, Metrics and Traces
 
 <figure><img src=".gitbook/assets/3.0.png" alt=""><figcaption></figcaption></figure>
 
-[Fluent Bit](http://fluentbit.io) is a Fast and Lightweight **Telemetry Agent** for Logs, Metrics, and Traces for Linux, macOS, Windows, and BSD family operating systems. It has been made with a strong focus on performance to allow the collection and processing of telemetry data from different sources without complexity.![](https://static.scarf.sh/a.png?x-pxid=71f0e011-761f-4c6f-9a89-38817887faae)
+[Fluent Bit](http://fluentbit.io) is a fast and lightweight **telemetry agent** for logs, metrics, and traces for Linux, macOS, Windows, and BSD family operating systems. It has been made with a strong focus on performance to allow the collection and processing of telemetry data from different sources without complexity.![](https://static.scarf.sh/a.png?x-pxid=71f0e011-761f-4c6f-9a89-38817887faae)
```

Comment:
This looks good. Could you fix the line wrapping to ~80 characters for the changes you're making in this PR? Thanks!

Reply:
yep, will do!